# example of train and validate parameters setting
# it is always better to write absolute paths of files needed
# you never know from which location you launch the process

resume: false

validation_frequency: 3

checking_rate: 2 # Hertz

pos_update_threshold: 0.005 # meters

step_resolution: 0.1

target_tolerance: 0.2

solver_config: /home/leonida/ThesisCode/LSTM_on_Create2/src/neural_network_planner/NetModels/LSTM/deep_lstm_solver.prototxt

trained_weights: /home/leonida/ThesisCode/LSTM_on_Create2/src/neural_network_planner/NetModels/LSTM/Snapshots/stack-fc/yaw/LSTM_deep-multiclass-3-64-16-27_iter_500000.caffemodel

GPU: true

total_iterations: 20000

time_sequence: 10

folder_path: /home/leonida/ThesisCode/LSTM_on_Create2/src/neural_network_planner/NetModels/

averaged_ranges_size: 12

multiclass: true

steer_feedback: false

steer_resolution: 15

min_steer_angle: -45
max_steer_angle: 45

logs_path: /home/leonida/ThesisCode/LSTM_on_Create2/logs/ 

dist_scale: 1

yaw_scale: 25

batch_updates: 3

iter_size: 3


